src/check_qemu_version.py

```
@@ -98,6 +96,16 @@ def execute(cmd):
     return content
 
 
+def parse_qemu_version(version):
+    r = re.compile(r'.*version\s([\d\.]+\([\w\d\.\s:\+-]+\)).*')
+    match = r.match(version)
+    # Return None if couldn't match string. Deal with None as couldn't 
+    # retrieve version.
+    if match is None:
+        return None
```

we should throw the exception right here i think